### ISSUE TITLE
Add the user manual pdf on github and translate it to english

### ISSUE THREAD

**Issue by GeorgeFkd** [open-source-uom/myuom#14]
There is currently a pdf hosted on our team's website, it would be helpful if it was on github(include both .docx and .pdf for further editing)

**Comment by GDGouravDey** [comment]
I have both these files, which folder or path should I include them in?

**Comment by GDGouravDey** [comment]
I am currently working on Translating the .docx file to English. But do I need to modify the Images with Greek Text too?

**Comment by GDGouravDey** [comment]
I have translated both the pdf and docx files to English. However, I have one doubt. I have translated all Greek text to English, but there are a few images where Greek text is used. If I want to change them to have English text instead, I would need to click new images of the pages in English. So, do I need to change the Greek Text in these Images to English?

Proof of Concept :-

![translate](https://github.com/open-source-uom/myuom/assets/117531461/c1ae81fe-36fb-4868-900e-78f7d40b8cab)
